packages/reports/tests/integration/components/print-report-view-test.js

```diff
@@ -1,131 +1,149 @@
     },
     {
       adClicks: 716731,
-      dateTime: '2015-11-14 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-14 00:00:00.000'
     },
     {
       adClicks: 399790,
-      dateTime: '2015-11-14 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-14 00:00:00.000'
     },
     {
       adClicks: 699490,
-      dateTime: '2015-11-14 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-14 00:00:00.000'
     }
   ]
-};
+});
 
 module('Integration | Component | print report view', function(hooks) {
   setupRenderingTest(hooks);
   setupMirage(hooks);
 
   hooks.beforeEach(async function() {
     const metadataService = this.owner.lookup('service:navi-metadata');
-    const store = this.owner.lookup('service:store');
 
     await metadataService.loadMetadata();
     this.set('response', RESPONSE);
 
     //set report object
-    this.set(
-      'report',
-      store.createRecord('report', {
-        request: store.createFragment('bard-request/request', {
-          logicalTable: store.createFragment('bard-request/fragments/logicalTable', {
-            table: metadataService.getById('table', 'spaceId', 'bardOne'),
-            timeGrain: 'day'
-          }),
-          responseFormat: 'csv',
-          intervals: A([{ interval: new Interval('current', 'next') }])
-        }),
-        visualization: {
-          type: 'line-chart',
-          version: 1,
-          metadata: {
-            axis: {
-              y: {
-                series: {
-                  type: 'metric',
-                  config: {
-                    metrics: [
-                      {
-                        metric: 'adClicks',
-                        parameters: {},
-                        canonicalName: 'adClicks',
-                        longName: 'Ad Clicks'
-                      }
-                    ]
-                  }
+    this.set('report', {
+      id: 13,
+      title: 'RequestV2 testing report',
+      createdOn: '2015-04-01 00:00:00',
+      updatedOn: '2015-04-01 00:00:00',
+      authorId: 'navi_user',
+      deliveryRuleIds: [],
+      visualization: {
+        type: 'line-chart',
+        version: 2,
+        metadata: {
+          style: {
+            area: false,
+            stacked: false
+          },
+          axis: {
+            y: {
+              series: {
+                type: 'dateTime',
+                config: {
+                  timeGrain: 'day',
+                  metricCid: 'c2'
                 }
               }
             }
           }
         }
-      })
-    );
+      },
+      request: {
+        table: 'network',
+        dataSource: 'bardOne',
+        limit: null,
+        requestVersion: '2.0',
+        filters: [
+          {
+            type: 'timeDimension',
+            dataSource: 'bardOne',
+            field: 'network.dateTime',
+            parameters: { grain: 'day' },
+            operator: 'bet',
+            values: ['11-04-2020', '11-06-2020']
```

Comment:
I think these should line up with the mock response
```suggestion
            values: ['2015-11-09', '2015-11-14']
```